web/modules/custom/hoeringsportal_content_blocks/src/Plugin/Block/MeetingWarning.php

```diff
@@ -12,16 +15,61 @@
  *   admin_label = @Translation("Node public meeting warning"),
  * )
  */
-class MeetingWarning extends BlockBase {
+class MeetingWarning extends BlockBase implements ContainerFactoryPluginInterface {
+
+  /**
+   * The node the block appears on.
+   *
+   * @var \Drupal\Core\Routing\CurrentRouteMatch
+   */
+  protected $currentNode;
+
+  /**
+   * Block constructor.
+   *
+   * @param array $configuration
+   *   Block configuration.
+   * @param string $plugin_id
+   *   Block plugin id.
+   * @param mixed $plugin_definition
+   *   Block plugin definition.
+   * @param \Drupal\Core\Routing\CurrentRouteMatch $routeMatch
+   *   The route match.
+   */
+  public function __construct(array $configuration, $plugin_id, $plugin_definition, CurrentRouteMatch $routeMatch) {
+    parent::__construct($configuration, $plugin_id, $plugin_definition);
+    $this->currentNode = $routeMatch->getParameter('node');
```

Comment:
You should not get the current node here, but store the service for later use, e.g.
```php
    $this->routeMatch = $routeMatch;
```